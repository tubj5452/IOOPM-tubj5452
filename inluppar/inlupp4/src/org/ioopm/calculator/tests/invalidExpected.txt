Welcome to the parser!
Please enter an expression: 
Error: Expected number
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Numbers cannot be used as a variable name
Please enter an expression: 
x + y
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Unexpected '+'
Please enter an expression: 
a + b
Please enter an expression: 
Error: Not a valid assignment of a variable
Please enter an expression: 
2.0 * x
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Unexpected '('
Please enter an expression: 
1.0 / 0.0
Please enter an expression: 
NaN
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Unexpected '+'
Please enter an expression: 
invalidCommand
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Unexpected '('
Please enter an expression: 
1.0 + x
Please enter an expression: 
Error, assignments to named constants:
3.2 = pi
42.0 = L
Please enter an expression: 
Error, the variables are reassigned: 
x
Please enter an expression: 
Error, the variables are reassigned: 
x
Please enter an expression: 
Variables: 
ans = 1.0 + x
c = a + b
d = a + b
y = 2.0 * x
z = 2.0 * x
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Expected number
Please enter an expression: 
expected '}'
Please enter an expression: 
3.0 + r
Please enter an expression: 
4.0
Please enter an expression: 
Invalid identifiers as arguments for conditional
Please enter an expression: 
Invalid identifiers as arguments for conditional
Please enter an expression: 
Error: Unexpected '}'
Please enter an expression: 
Error: Unexpected '<'
Please enter an expression: 
Error: Expected number
Please enter an expression: 

You have quit the program
Total entered expression(s): 33
Successful evaluated expression(s): 9
Expression(s) that were fully evaluated: 2
