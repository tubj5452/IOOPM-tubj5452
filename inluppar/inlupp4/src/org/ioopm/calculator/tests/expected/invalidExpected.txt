Welcome to the parser!
Please enter an expression: 
Error: Expected number
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Numbers cannot be used as a variable name
Please enter an expression: 
x + y
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Unexpected '+'
Please enter an expression: 
a + b
Please enter an expression: 
Error: Not a valid assignment of a variable
Please enter an expression: 
2.0 * x
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Unexpected '('
Please enter an expression: 
1.0 / 0.0
Please enter an expression: 
NaN
Please enter an expression: 
expected ')'
Please enter an expression: 
Error: Unexpected '+'
Please enter an expression: 
invalidCommand
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Unexpected '('
Please enter an expression: 
1.0 + x
Please enter an expression: 
Error, assignments to named constants:
3.2 = pi
42.0 = L
Please enter an expression: 
Error, the variables are reassigned: 
x
Please enter an expression: 
Error, the variables are reassigned: 
x
Please enter an expression: 
Variables: 
ans = 1.0 + x
c = a + b
d = a + b
y = 2.0 * x
z = 2.0 * x
Please enter an expression: 
Error: Expected number
Please enter an expression: 
Error: Expected number
Please enter an expression: 
expected '}'
Please enter an expression: 
3.0 + r
Please enter an expression: 
4.0
Please enter an expression: 
Invalid identifiers as arguments for conditional
Please enter an expression: 
Invalid identifiers as arguments for conditional
Please enter an expression: 
Error: Unexpected '}'
Please enter an expression: 
Error: Unexpected '<'
Please enter an expression: 
Error: Expected number
Please enter an expression: 
expected ')'
Please enter an expression: 
expected ')' after function declaration
Please enter an expression: 
Function needs at least one parameter
Please enter an expression: 
Please enter an expression: 
Cannot redefine function
Please enter an expression: 
Cannot redefine function
Please enter an expression: 
Error: Function 'max' called with the wrong number of arguments. Expected 2, got 3
Please enter an expression: 
Error: Function 'max' called with the wrong number of arguments. Expected 2, got 1
Please enter an expression: 
Assignment in function can not have the same name as param
Please enter an expression: 

You have quit the program
Total entered expression(s): 42
Successful evaluated expression(s): 11
Expression(s) that were fully evaluated: 2
